homeassistant/components/systemmonitor/sensor.py

```diff
@@ -130,12 +131,14 @@ def get_throughput(entity: SystemMonitorSensor[dict[str, snetio]]) -> float | No
     state = None
     if entity.argument in counters:
         counter = counters[entity.argument][IO_COUNTER[entity.entity_description.key]]
-        now = dt_util.utcnow()
-        if entity.value and entity.update_time and entity.value < counter:
+        now = time.time()
```

Comment:
```suggestion
        now = time.monotonic()
```

Reply:
Any particular reason why `time.monotonic()` gives `1970-01-02 20:51:52.875012` on first update and not current date/time timestamp?

Reply:
Sounds weird unless the next call is close to 1970-01-02. Only time comparisons are valid when using `time.monotonic`.

Reply:
Next call returns correct date/time of today so have no idea. Wonder if there's something strange here locally.

Reply:
This is in my environment:

```
Python 3.11.7 (main, Dec  8 2023, 18:56:57) [GCC 9.4.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.14.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import time

In [2]: test_1 = time.monotonic()

In [3]: test_1
Out[3]: 1607048.847247169

In [4]: test_2 = time.monotonic()

In [5]: test_2
Out[5]: 1607063.008190904
```

Reply:
Ok, so on first pass it returns correctly the seconds from 1970-xx...
On second pass it has moved to today's date so obviously the minute change I want is becoming much more.
It's all happening once I move the time forward and from what I can see `hass.loop.time()` has also exactly same behavior so it seems not limited to this test alone.

Looks like we patch the monotonic clock for freezegun (see  somehow so it's no longer seconds from epoch but it just starts behaving as the `time.time()`
Not sure how to move forward from here.

Reply:
In the bluetooth tests we have to re-patch `time.monotonic` ourselves because of freezegun

Reply:
So I changed the order of the mockers so freezer goes first. Then all is fine as we have monotonic time wrong from before loading the integration. So no need to patch further.